### PR TITLE
Update 10min_to_cudf_colab.ipynb

### PR DESCRIPTION
**Pull request by taureandyernv**
updated pip install to current template

### CODE REVIEW

**Comment by GregoryKimball** [comment]
Would you please add an `import os` command to the "Importing and Getting Started" block?
image

I ran the notebook from this PR with this link: https://colab.research.google.com/github/rapidsai-community/showcase/blob/update_colab_pip/getting_started_tutorials/10min_to_cudf_colab.ipynb#scrollTo=4c6c548b

**Review by GregoryKimball — APPROVED** [review]
Looks good. This runs to completion now. Thank you!

File: getting_started_tutorials/10min_to_cudf_colab.ipynb [hunk 1]
```diff
@@ -90,24 +92,48 @@
...
+        "1. Checks to make sure that the GPU is RAPIDS compatible\n",
+        "1. Installs the **current stable version** of RAPIDSAI's core libraries using pip, which are:\n",
+        "  1. cuDF\n",
+        "  1. cuML\n",
+        "  1. cuGraph\n",
+        "  1. xgboost\n",
+        "\n",
+        "**This will complete in about 3-4 minutes**\n",
+        "\n",
+        "Please use the [RAPIDS Conda Colab Template notebook](https://colab.research.google.com/drive/1TAAi_szMfWqRfHVfjGSqnGVLr_ztzUM9) if you need to install any of RAPIDS Extended libraries, such as:\n",
```
Comment (beckernick) [inline getting_started_tutorials/10min_to_cudf_colab.ipynb:110]:
```suggestion
        "Please use the [RAPIDS Conda Colab Template notebook](https://colab.research.google.com/drive/1TAAi_szMfWqRfHVfjGSqnGVLr_ztzUM9) if you need to install any of the RAPIDS Extended libraries, such as:\n",
```

File: getting_started_tutorials/10min_to_cudf_colab.ipynb [hunk 2]
```diff
@@ -90,24 +92,48 @@
...
+        "id": "43lj_9YlKxx6"
       },
       "outputs": [],
       "source": [
-        "!pip install cudf-cu11 --extra-index-url=https://pypi.ngc.nvidia.com\n",
-        "\n",
-        "#Older documentation included these steps to reintall cupy, but this is no longer necessary since Rapids 22.12\n",
-        "#!rm -rf /usr/local/lib/python3.8/dist-packages/cupy*\n",
-        "#!pip install cupy-cuda11x"
+        "# This get the RAPIDS-Colab install files and test check your GPU.  Run this and the next cell only.\n",
```
Comment (beckernick) [inline getting_started_tutorials/10min_to_cudf_colab.ipynb:133]:
```suggestion
        "# This gets the RAPIDS-Colab install files and test check your GPU.  Run this and the next cell only.\n",
```

**Review by beckernick — APPROVED** [review]
Couple of minor suggestions. Thanks for updating this!

**Review by GregoryKimball — APPROVED** [review]

**Pull request merged**